### ISSUE TITLE
5.22: UNOP_AUX multideref

### ISSUE THREAD

**Issue by rurban** [rurban/perl-compiler#215]
Add a `C.xs` API to get the aux buffer as PV, maybe with the preceding length field, which is skipped in the official API.
We need that for the Bytecode and C compiler.

This buffer needs to be patched with updated GV, PAD and SV pointers. The PADs and SVs can be patched up at compile-time, but GVs need to be patched at init-time.

See the `bc522` branch

**Comment by atoomic** [comment]
this is probably an illustration of the problem

Environment:
- perl 5.22
- bc522 branch at commit 23016e1

````
> perl -e 'my %h; $h{aa}; print qq{ok\n}'
ok
> perlcc -r -e 'my %h; $h{aa}; print qq{ok\n}'
````

**Comment by rurban** [comment]
Fixed in bc522 with c5e2c899371940e42fb62f78c8daa6dff1b52cc6

**Comment by rurban** [comment]
oops, not yet fully fixed. 
-1 iv and pad_offsets are stored as SV, not as constants.
        # TODO pad_offset, iv detection. store with proper type.

**Comment by rurban** [comment]
Fixed with 13faf89ca67edeaf6aec4668512c6eaee2780ddb
